### ISSUE TITLE
I tried to make a quizzer and there was an error

### ISSUE THREAD

**Issue by danielboniface** [denverpost/quizzer#5]
Can anyone tell me what went wrong here? @schneidan @freejoe76 @ericjlubbers 

http://www.denverpost.com/broncos/ci_29003161/

**Comment by schneidan** [comment]
First, the embedded ad landed in the middle of the script. I turned it off.

Second, the URL for quiz.js wasn't valid. I replaced it with:

```
<script type="text/javascript" src="http://extras.denverpost.com/media/js/quiz.0.5.js"></script>
```

Seems to work now: http://www.denverpost.com/broncos/ci_29003161/quiz-how-many-denver-broncos-ring-fame-inductees

@freejoe76 Is there anything we can do to make the stuff that goes in the article more compact and avoid hitting the embedded ad? I suppose we could just add "Turn off embedded ads in the article" to the instructions...
